src/connectRoutes.js

```diff
@@ -115,7 +115,11 @@ export default (
   }
 
   const {
+    // Flow doesn't like defaulting to string or func here, although it does not
+    // complain if we _do not_ provide a default value
+    // $FlowIssue
```

Comment:
This is not pretty, but flow was not having a good time with default arguments here. However, flow seems to know the type but type this incorrectly. Maybe I'm missing something, but flow seemed to be saying two conflicting things at once:

```
src/connectRoutes.js:118
118:     location: locationKey = 'location',
                   ^^^^^^^^^^^ function type. This type is incompatible with
118:     location: locationKey = 'location',
                   ^^^^^^^^^^^ string

src/connectRoutes.js:118
118:     location: locationKey = 'location',
                   ^^^^^^^^^^^ string. This type is incompatible with
118:     location: locationKey = 'location',
                   ^^^^^^^^^^^ function type
```